content/2015.12.decentralised-en.rst

```diff
@@ -0,0 +1,138 @@
+the hard promess of self-hosting
+################################
+
+:date: 2015-12-22
+:slug: difficile-promesse-autohebergement
+:lang: fr
+:url: self-hosting-a-hard-promess
+:summary: Where we talk about self-hosting, deployment and decentralization.
+
+.. note::
+
+  This article has been translated by
```

Comment:
This article was translated by Alexis Métaireau on a train between Rennes and Paris.